javascript/javascript2/week3/functionAsVariable/functionAsVariable.js

```diff
@@ -0,0 +1,24 @@
+//Create an array with 3 items. All items should be functions. Iterate through the array and call all the functions.
+const arrayOfFunctions = [
+    () => console.log("Called first function"),
+    () => console.log("Called second function"),
+    () => console.log("Called third function")
+];
+
+arrayOfFunctions.forEach(element => element());
+
+//Create a function as a const and try creating a function normally. Call both functions.
+ const name = function nameFinder(name){
```

Comment:
you don't need to use the named function form here

```
const name = function (name) {
...
```
that would have also worked